### ISSUE TITLE
Ability to set/force Aspect Ratio per feed

### ISSUE THREAD

**Issue by alanmcramer** [Fleker/CumulusTV#108]
I have some feeds that happen to be 4:3 but when using Cumulus TV, it forces these to be widescreen.  Problem is everyone looks like they have gained weight.  Would be great if in the edit channel screen, and in the json file there was a way to be able to force a certain channel to show as 4:3 with black bars either side or 16:9.  Maybe have a few other aspect ratios as well for good measure.

In the Settings for the app itself, one could change the colour of the black bars to grey or white.

This change may be one that requires changes to ChannelSurfer if this is possible.

**Comment by Fleker** [comment]
Yeah I know it's an issue. I'm not sure how to solve it at the moment.
There's probably a way to do this automatically, but for now it's forced to
16:9.

**Comment by alanmcramer** [comment]
That's why there should be a way within each channel to force an aspect ratio.  Even if this was implemented on the backend only, that you can add the setting within the JSON instead of needing to build a GUI for it.

  {
            "number": "1337", //Channel Number
            "name": "Aquarium", //Channel Title
            "logo": "logo.jpg", //Channel Logo
            "url":"stream.m3u8", //Channel stream or website url
            "splashscreen":"" //Optional splashscreen instead of default one
            "aspect": "4:3" //Sets Aspect Ratio
            "genres":"NEWS,MUSIC" //Comma separated array of channel genres  
        }

**Comment by Fleker** [comment]
I know it could be set manually as a last resort, having the video player
know the correct ratio without extra work would be ideal.

**Comment by Fleker** [comment]
I should probably remove the forced aspect ratio and let the player do what makes sense.

**Comment by alanmcramer** [comment]
If that option exists.... Otherwise implementing something within the json
for each channel would work also. I believe aspect ratio is part of the
xmltv spec.

**Comment by nextux** [comment]
I've tested some of my streams with TIF, and found it correctly identifies the stream as 4:3 or 16:9.  Assuming this love will be in the next release of Cumulus TV as it will have the same library attached now?  If so, this may not be required.

**Comment by Fleker** [comment]
If so, then it should work as expected

**Comment by nextux** [comment]
Seems 1.7.1 is still forcing videos to 16:9 when the source video is 4:3.  Unsure if there is some code behind the scenes in Cumulus that does this.  As mentioned when using the TIF demo and using the exact same m3u8 stream, it correctly shows the 4:3 videos in 4:3 aspect ratio with black bars left and right.

**Comment by Fleker** [comment]
I've been looking into it. I think I'm doing something wrong that's causing it to stretch out.

**Comment by nicco** [comment]
I think you should also keep, among other modes, the current mode too!
